cfme/tests/test_manual.py

```diff
@@ -3035,23 +2934,73 @@ def test_control_import_with_incorrect_schema():
 
 
 @pytest.mark.manual
-@test_requirements.rest
-def test_cloud_volume_types():
+@test_requirements.log_depot
+@pytest.mark.tier(2)
+def test_log_azure_settings_azure():
```

Comment:
Please remove this test, it's automated
https://github.com/ManageIQ/integration_tests/blob/master/cfme/tests/configure/test_logs.py#L114